app/helpers/question_helper.rb

```diff
@@ -1,2 +1,9 @@
 module QuestionHelper
+  def question_header(object)
```

Comment:
лучше не скрывать сущность переменной )
```suggestion
  def question_header(question)
```